WallPaper_Project1/WallPaper_Project1Tests/ImageScreenTest.swift

```diff
@@ -0,0 +1,123 @@
+                  photographerId: 174849153,
+                  avgColor: "#28261A",
+                  source: WallPaper_Project1.Source(
+                    original: "https://images.pexels.com/photos/15456966/pexels-photo-15456966.jpeg",
+                    large2x: "https://images.pexels.com/photos/15456966/pexels-photo-15456966.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
+                    large: "https://images.pexels.com/photos/15456966/pexels-photo-15456966.jpeg?auto=compress&cs=tinysrgb&h=650&w=940",
+                    medium: "https://images.pexels.com/photos/15456966/pexels-photo-15456966.jpeg?auto=compress&cs=tinysrgb&h=350",
+                    small: "https://images.pexels.com/photos/15456966/pexels-photo-15456966.jpeg?auto=compress&cs=tinysrgb&h=130",
+                    portrait: "https://images.pexels.com/photos/15456966/pexels-photo-15456966.jpeg?auto=compress&cs=tinysrgb&fit=crop&h=1200&w=800",
+                    landscape: "https://images.pexels.com/photos/15456966/pexels-photo-15456966.jpeg?auto=compress&cs=tinysrgb&fit=crop&h=627&w=1200",
+                    tiny: "https://images.pexels.com/photos/15456966/pexels-photo-15456966.jpeg?auto=compress&cs=tinysrgb&dpr=1&fit=crop&h=200&w=280"),
+                  liked: false,
+                  alt: ""),
+            Image(id: 15464741,
+                  width: 3024,
+                  height: 4032,
+                  url: "https://www.pexels.com/photo/bread-food-plate-restaurant-15464741/",
+                  photographer: "Yelena Odintsova",
+                  photographerUrl: "https://www.pexels.com/@yelenaodintsova",
+                  photographerId: 3068556,
+                  avgColor: "#A49F99",
+                  source: WallPaper_Project1.Source(
+                    original: "https://images.pexels.com/photos/15464741/pexels-photo-15464741.jpeg",
+                    large2x: "https://images.pexels.com/photos/15464741/pexels-photo-15464741.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940",
+                    large: "https://images.pexels.com/photos/15464741/pexels-photo-15464741.jpeg?auto=compress&cs=tinysrgb&h=650&w=940",
+                    medium: "https://images.pexels.com/photos/15464741/pexels-photo-15464741.jpeg?auto=compress&cs=tinysrgb&h=350",
+                    small: "https://images.pexels.com/photos/15464741/pexels-photo-15464741.jpeg?auto=compress&cs=tinysrgb&h=130",
+                    portrait: "https://images.pexels.com/photos/15464741/pexels-photo-15464741.jpeg?auto=compress&cs=tinysrgb&fit=crop&h=1200&w=800",
+                    landscape: "https://images.pexels.com/photos/15464741/pexels-photo-15464741.jpeg?auto=compress&cs=tinysrgb&fit=crop&h=627&w=1200",
+                    tiny: "https://images.pexels.com/photos/15464741/pexels-photo-15464741.jpeg?auto=compress&cs=tinysrgb&dpr=1&fit=crop&h=200&w=280"),
+                  liked: false,
+                  alt: "")
+        ]
+        XCTAssert(images.isEmpty, "nil")
```

Comment:
tương tự mấy cái khác e
```suggestion
        XCTAssertEqual(images.isEmpty, false)
```